tumbleweed

```diff
@@ -209,7 +209,12 @@ tumbleweed_uninit()
 tumbleweed_install()
 {
   sudo zypper ref
-  sudo zypper dup
+
+  if [ "$no_allow_vendor_change" == "1" ] ; then
+    sudo zypper dup --no-allow-vendor-change
```

Comment:
Isn't no allow vendor change the default in Tumbleweed or was that changed back? I am not sure we want to start adding all zypper flags here. Also with [properly set repository priorities](https://www.youtube.com/watch?v=x8kEaJU6hlw) one wants to allow vendor change.

Essentially, if I want to do something fancy like `tumbleweed switch` and `zypper dup -d` I do those commands separately. I am not sure we want to start supporting a variety of use-cases of zypper.

Reply:
It appears to have been rolled back on my fresh openSUSE install, but I understand the implication of needing to add multiple zypper flags. I'll leave it up to you if you'd like to accept this PR or not, otherwise I can close it.